trimmer_decorator.rb

```diff
@@ -0,0 +1,8 @@
+require './base_decorator'
+class TrimmerDecorator < baseDecorator
+  def correct_name
+    return @nameable.correct_name[0..9] if @nameable.correct_name.length < 10
+
+    @nameable.correct_name
```

Comment:
- [ ] Kindly make sure the `correct_name` method can return the name that has a maximum of **10 characters**. If it's longer it should trim the word. Make sure you update your condition to be **>** and not **<**